src/jit/codegenarm64.cpp

```diff
@@ -2470,7 +2470,8 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                 // We unconditionally throw a divide by zero exception
                 genJumpToThrowHlpBlk(EJ_jmp, SCK_DIV_BY_ZERO);
 
-                // We still need to call genProduceReg
+                // We still need to call genConsumeOperands/genProduceReg for liveness update.
+                genConsumeOperands(treeNode->AsOp());
```

Comment:
In view of consistency we can move out genConsumeOperands() call from xarch version as well.  There is a single call to genCodeForBinary() in codegenxarch.cpp.

To me it is unnecessary overhead to add an additional param to indicate whether to consume operands or not.